autopush/logging.py

```diff
@@ -118,6 +118,10 @@ def raven_log(self, event):
             exc_info=(f.type, f.value, tb),
             stack=stack,
             extra=extra,
+            log_format=event.get('log_format'),
```

Comment:
AFAICT raven's pretty much going to ignore these kwargs. You wanna put them into the extra dict, that's its really where it wants arbitrary kwargs that'll show on the UI